aiida/tools/importexport/dbimport/backends/django/__init__.py

```diff
@@ -690,8 +690,6 @@ def import_data_dj(
                 print('NO NODES TO IMPORT, SO NO GROUP CREATED, IF IT DID NOT ALREADY EXIST')
 
     if not silent:
-        print('*** WARNING: MISSING EXISTING UUID CHECKS!!')
```

Comment:
Fine for me to remove these messages.
I presume the second comment are actually thought non-existing features that should have been added. But I do not consider this the right place to put those suggestions. And as they are not very well described, it is fine to remove.
The first comment is just wrong.

Note, in #3599 I introduce the `debug` parameter, where messages such as these should go.